LocalSettings.php

```diff
@@ -804,31 +804,41 @@ function wfShouldEnableSwift( $dbname ) {
-				'Duplicate wiki' => 'Your proposed wiki appears to duplicate, either substantially or entirely, the content of an existing wiki (see the "Request Comments" tab for one or more link(s) to the existing wiki(s)). Could you please describe in a few more sentences by adding to, but not replacing, your existing description, the scope and focus for your wiki, and also assure us that your wiki will not be a complete or substantial duplication? Thank you.',
+				'Content Policy (makes it difficult for other wikis)' => 'Declining per Content Policy provision, "A wiki must not create problems which make it difficult for other wikis." Thank you for understanding.',
+				'Content Policy (no anarchy wikis)' => 'Declining per Content Policy provision, "Miraheze does not host wikis that operate on the basis of an anarchy system (i.e. no leadership and no rules)." Thank you for understanding.',
+				'Content Policy (sexual nature involving minors)' => 'Declining per Content Policy provision, "Miraheze does not host wikis of a sexual nature which involve minors in any way." Thank you for your understanding.',
+				'Content Policy (toxic communities)' => 'Declining per Content Policy provision, "Miraheze does not host wikis where the community has developed in such a way as to be characterized as toxic." Thank you for your understanding.',
+				'Content Policy (unsubstantiated insult)' => 'Declining per Content Policy provision, "Miraheze does not host wikis with the sole purpose to spread unsubstantiated insult, hate or rumours against a person or group of people." Thank you for understanding.',
+				'Content Policy (violence or hatred)' => 'Declining per Content Policy provision, "Miraheze does not host wikis that promote violence, hatred, or harassment against a person or group of people." Thank you for your understanding.',
+				'Content Policy (Wikimedia-like wikis/forks)' => 'Declining per Content Policy provision, "Direct forks and forks where a substantial amount of content is copied from a Wikimedia project are also not allowed." Thank you for your understanding.',
 				'Author request' => 'Declined at the request of the wiki requester.',
 			],
 			'On hold reasons' => [
 				'On hold pending response' => 'On hold pending response from the wiki requester (see the "Request Comments" tab). Please reply to the questions left by the wiki creator on this request but do not create another wiki request. Thank you.',
-				'On hold pending review' => 'On hold pending review from another wiki creator or steward. Please do not resubmit this request.',
+				'On hold pending review from another wiki creator' => 'On hold pending review from another Wiki creator or Steward. Please do not resubmit this request.',
```

Comment:
```suggestion
				'On hold pending review from another wiki creator' => 'On hold pending review from another Wiki creator or Steward.',
```
Although I fixed wiki requests being edited now, editing an on-hold request does not actually mark it as inreview again, as I didn't think it should, it should remain on hold until wiki creators action it. But if we wish to change that we also can.